modules/installation-launching-installer.adoc

```diff
@@ -230,6 +231,9 @@ ifeval::["{context}" == "installing-nutanix-installer-provisioned"]
 :nutanix:
 :single-step:
 endif::[]
+ifeval::["{context}" == "installing-ibm-power-vs-customizations"]
+:custom-config:
```

Comment:
@alishaIBM I believe you need to add the `single-step` attribute here. After you do, check your published output. You should have a single bulleted step for running the installation program with an existing `install-config.yaml` file.